packages/trader/src/AppV2/Components/Guide/guide.scss

```diff
@@ -0,0 +1,164 @@
+.guide {
+    &__title {
+        margin-block-end: var(--component-actionSheet-spacing-padding-lg);
+    }
+
+    &__menu {
+        padding-inline-end: var(--component-actionSheet-spacing-padding-lg);
+        margin-block-end: var(--component-actionSheet-spacing-padding-md);
+        display: flex;
+        gap: var(--core-spacing-400);
+        overflow-x: auto;
+        min-height: var(--core-spacing-1600);
+        white-space: nowrap;
+        -ms-overflow-style: none;
+        scrollbar-width: none;
+    }
+
+    &__wrapper {
+        padding: var(--component-actionSheet-spacing-padding-lg) var(--component-actionSheet-spacing-padding-lg) 0;
+    }
+
+    &__contract-description {
+        overflow-y: auto;
+        height: calc(90vh - 23rem);
+    }
+
+    &__button {
+        padding: var(--component-actionSheet-spacing-padding-lg);
+    }
+
+    &-video {
+        position: relative;
+        border-radius: var(--core-borderRadius-400);
+
+        &__preview {
+            position: relative;
+            border-radius: var(--core-borderRadius-400);
+
+            &__icon {
+                position: absolute;
+                top: 50%;
+                left: 53%;
```

Comment:
we can replace all 'left's here with 'inset-inline-start's :)

Reply:
Sure)